pixeltable/utils/code.py

```diff
@@ -0,0 +1,27 @@
+import types
```

Comment:
code.py?

Might be a good idea to leave a comment describing what should go in here.

Reply:
Sure, I'll do that. I'm also open to suggestions for a better name for the .py file :/